app/components/resend-invitation/component.js

```diff
@@ -0,0 +1,24 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  tagName: '', // tagless component
+
+  store: Ember.inject.service(),
```

Comment:
components already have the store via:

https://github.com/aptible/diesel.aptible.com/blob/master/app/initializers/setup-component-with-store.js#L2

this injection would fail if you used `this.get('store')` as the store is not a service (not in the container as such). The lazy injections with `Ember.inject` need to be fetched via `.get`, and since you used `this.store` below this injection is actually never used.